MongoScript/seed2.js

```diff
@@ -23,15 +23,17 @@ let items = (id) => {
   });
 }
 
+let id_gen = 3340001;
 let create = async () => {
```

Comment:
similar to before i think this could have a more descriptive name, i.e. `create`...Something. what is it creating? and what does the `items()` function do? it's not entirely clear from the names

Reply:
i'll actively work on better and more descriptive variable and function names.